bodhi/server/models.py

```diff
@@ -1348,17 +1384,22 @@ class Build(Base):
     """
 
     __tablename__ = 'builds'
-    __exclude_columns__ = ('id', 'package', 'package_id', 'release',
-                           'update_id', 'update', 'override')
+    __exclude_columns__ = ('id', 'package', 'release', 'update', 'override')
     __get_by__ = ('nvr',)
+    __relation_fields__ = ('id', 'nvr', 'type')
```

Comment:
I would probably add the signed field here since it is quite important info to have about a build

Reply:
Field added